lib/engine/game/g_18_usa/step/buy_pullman.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require_relative '../../../step/base'
+require_relative '../../../step/train'
+module Engine
+  module Game
+    module G18USA
+      module Step
+        class BuyPullman < Engine::Step::Base
```

Comment:
Rather than reimplement much of the buy train step, is it possible to use buy train, but limit to just pullmans?

Reply:
If its because of the repay loan assuming buy train happens only right before repay loan, then fix that instead of duplicating logic. You can fix it by setting a round variable or something along those lines.

Reply:
This inherits from BuyTrain now and I added a check to 18USA's operating round to get around the loan interest issue